app/src/main/java/net/gsantner/markor/util/StringUtils.java

```diff
@@ -0,0 +1,67 @@
+package net.gsantner.markor.util;
+
+import org.apache.commons.lang3.ArrayUtils;
+
+public class StringUtils extends org.apache.commons.lang3.StringUtils {
+    // modified version of org.apache.commons.lang3.StringUtils#indexOfAny
+    /**
+     * Same as {@link org.apache.commons.lang3.StringUtils#indexOfAny(CharSequence, char...)
+     * where you can specify the search intervall}
+     */
+    public static int indexOfAny(final CharSequence cs, int csFirst, int csLen, final char... searchChars) {
```

Comment:
We have [GsTextUtils](https://github.com/gsantner/markor/blob/master/app/src/main/java/net/gsantner/opoc/format/GsTextUtils.java) and [TextViewUtils](https://github.com/gsantner/markor/blob/master/app/src/main/java/net/gsantner/markor/frontend/textview/TextViewUtils.java)

These should go in one of them.

Reply:
I was not aware that these exist. Moved indexOfAny from [net.gsantner.markor.util.StringUtils] to [net.gsantner.opoc.format.GsTextUtils]